devtools/startup/DevToolsStartup.jsm

```diff
@@ -739,23 +740,23 @@ DevToolsStartup.prototype = {
   },
 
   initializeRecordingWebChannel() {
-    const pageUrl = Services.prefs.getStringPref(
-      "devtools.recordreplay.recordingsUrl"
-    );
-    const localUrl = "http://localhost:8080/view";
-
-    registerWebChannel(pageUrl);
-    registerWebChannel(localUrl);
-
-    function registerWebChannel(url) {
-      const urlForWebChannel = Services.io.newURI(url);
-      const channel = new WebChannel("record-replay-token", urlForWebChannel);
-
-      channel.listen((id, message) => {
-        const { token } = message;
-        saveRecordingToken(token);
-      });
-    }
+    // const pageUrl = Services.prefs.getStringPref(
+    //   "devtools.recordreplay.recordingsUrl"
+    // );
+    // const localUrl = "http://localhost:8080/view";
+
+    // registerWebChannel(pageUrl);
+    // registerWebChannel(localUrl);
+
+    // function registerWebChannel(url) {
+    //   const urlForWebChannel = Services.io.newURI(url);
+    //   const channel = new WebChannel("record-replay-token", urlForWebChannel);
+
+    //   channel.listen((id, message) => {
+    //     const { token } = message;
+    //     saveRecordingToken(token);
+    //   });
+    // }
```

Comment:
It's great to see this go away

Reply:
We _might_ have to bring it back in the opposite direction to inject auth into the app when running in the Replay browser but at least the browser will own it's own authentication process now.